tests/unit/fixtures/gameState.js

```diff
@@ -55,11 +59,210 @@ const fixture = {
+    turn: 1,
+    phase: GamePhase.MAIN,
+    p0: {
+      hand: [Card.TEN_OF_HEARTS],
+      points: [],
+      faceCards: [],
+    },
+    p1: {
+      hand: [Card.TEN_OF_DIAMONDS],
+      points: [Card.ACE_OF_HEARTS],
+      faceCards: [],
+    },
+    deck: [Card.SIX_OF_SPADES, Card.TWO_OF_HEARTS, Card.ACE_OF_CLUBS],
+    scrap: [],
+    oneOff: null,
+    oneOffTarget: null,
+    twos: [],
+    resolving: null,
+    gameId: 9,
+  },
+
+  gameStateRow: {
+    playedBy: 0,
+    moveType: MoveType.RESOLVE_THREE,
+    playedCard: '3S',
+    targetCard: 'TH',
+    discardedCards: [],
+    turn: 1,
+    phase: GamePhase.MAIN,
+    p0Hand: ['TH'],
+    p0Points: [],
+    p0FaceCards: [],
+    p1Hand: ['TD'],
+    p1Points: ['AH'],
+    p1FaceCards: [],
+    deck: ['6S', '2H', 'AC'],
+    scrap: [],
+    oneOff: null,
+    oneOffTarget: null,
+    twos: [],
+    resolving: null,
+    gameId: 9,
+  },
+
+  socket: {
+    game: {
+      players: [
+        {
+          hand: [Card.TEN_OF_HEARTS],
+          points: [],
+          faceCards: [],
+          createdAt: '2024-07-14T12:59:45.813Z',
+          updatedAt: '2024-07-14T12:59:51.528Z',
+          id: 17,
+          username: 'myUsername',
+          pNum: 0,
+          rank: 1000,
+          game: 9,
+          frozenId: null,
+        },
+        {
+          hand: [Card.TEN_OF_DIAMONDS],
+          points: [Card.ACE_OF_HEARTS],
+          faceCards: [],
+          createdAt: '2024-07-14T12:59:47.709Z',
+          updatedAt: '2024-07-14T12:59:51.769Z',
+          id: 18,
+          username: 'definitelyNotTheGovernment6969',
+          pNum: 1,
+          rank: 1000,
+          game: 9,
+          frozenId: null,
+        },
+      ],
+      deck: [Card.ACE_OF_CLUBS],
+      scrap: [],
+      twos: [],
+      spectatingUsers: [],
+      name: 'Test Game',
+      status: GameStatus.STARTED,
+      p0Ready: true,
+      p1Ready: true,
+      p0Rematch: null,
+      p1Rematch: null,
+      rematchGame: null,
+      passes: 0,
+      oneOffTargetType: '',
+      turn: 1,
+      log: ['myUsername took the 10♥️ from the Scrap pile to their hand.'],
+      turnStalemateWasRequestedByP0: -1,
+      turnStalemateWasRequestedByP1: -1,
+      chat: [],
+      isRanked: false,
+      pNum: 0,
+      playedBy: 0,
+      discardedCards: [],
+      chosenCard: Card.TEN_OF_HEARTS,
+      lastEvent: {
+        change: MoveType.RESOLVE_THREE,
+        pNum: 0,
+        chosenCard: Card.TEN_OF_HEARTS,
+        oneOffRank: null,
+        oneOffTargetType: '',
+        discardedCards: null,
+      },
+      lock: null,
+      lockedAt: null,
+      match: null,
+      winner: null,
+      topCard: Card.SIX_OF_SPADES,
+      secondCard: Card.TWO_OF_HEARTS,
+      oneOff: null,
+      resolving: null,
+      oneOffTarget: null,
+    },
+    victory: {
+      conceded: false,
+      currentMatch: null,
+      gameOver: false,
+      winner: null,
+    },
+  },
+};
+
+export { fixture, resolveThree };
```

Comment:
Can we rename the `fixture` variable after its moveType to match how you're doing resolveThree?